pkg/og/convert/pprof/profile.go

```diff
@@ -259,10 +109,14 @@ func (p *RawProfile) loadPprofFromForm() error {
 	if err != nil {
 		return err
 	}
-	p.PreviousProfile, err = form.ReadField(f, formFieldPreviousProfile)
+	PreviousProfile, err := form.ReadField(f, formFieldPreviousProfile)
 	if err != nil {
 		return err
 	}
+	if PreviousProfile != nil {
+		return fmt.Errorf("cumulative profile is nolonger supported. " +
+			"Update clients to the latest versions of pyroscope-golang/godeltaprof")
+	}
```

Comment:
I guess `godeltaprof` is our internal dependency users might not be aware of

```suggestion
	if PreviousProfile != nil {
		return fmt.Errorf("unsupported client version. " +
			"Please update github.com/grafana/pyroscope-go to the latest version")
	}
```